creusot/tests/should_succeed/sum.stdout

```diff
@@ -28,7 +28,7 @@ module Sum_SumFirstN
   clone CreusotContracts_Builtins_Resolve as Resolve1 with type self = bool
   clone CreusotContracts_Builtins_Resolve as Resolve0 with type self = uint32
   let rec cfg sum_first_n (n : uint32) : uint32
-    ensures { result = div (n * n + (1 : uint32)) (2 : uint32) }
+    ensures { result = div (n * (n + (1 : uint32))) (2 : uint32) }
```

Comment:
huh, that explains things: the printer was outputting the wrong expressions before!